aspnet/web-forms/overview/getting-started/hands-on-labs/whats-new-in-aspnet-and-web-development-in-visual-studio-2012.md

```diff
@@ -75,12 +75,12 @@ Visual Studio now includes features that help developers access, work and organi
 In this task, you will discover the new features of the CSS Editor. This new editor will help you increase your productivity by taking advantage of the new smart indentation, the improved code comments and the enhanced IntelliSense list.
 
 1. Start **Visual Studio** and open the **WhatsNewASPNET.sln** solution located in the **Source\WhatsNewASPNET** folder of this lab.
-2. In Solution Explorer, open the **Site.css** file located under the **Styles** folder. Make sure the **Text Editor** tools are visible on the toolbar. To do that, select the **View** | **Toolbars** menu option, and check the **Text Editor** options. You will notice that, since this new version, the **Comment** button (![comment-button](whats-new-in-aspnet-and-web-development-in-visual-studio-2012/_static/image1.png) ) and the **Uncomment** button (![uncomment-button](whats-new-in-aspnet-and-web-development-in-visual-studio-2012/_static/image2.png) ) are also enabled for the CSS editor.
+2. In Solution Explorer, open the **Site.css** file located under the **Styles** folder. Make sure the **Text Editor** tools are visible on the toolbar. To do that, select the **View** | **Toolbars** menu option, and check the **Text Editor** options. You will notice that, since this new version, the **Comment** button (:::image type="icon" source="whats-new-in-aspnet-and-web-development-in-visual-stuido-2012/_static/image1.png":::) and the **Uncomment** button (:::image type="icon" source="whats-new-in-aspnet-and-web-development-in-visual-stuido-2012/_static/image2.png":::) are also enabled for the CSS editor.
```

Comment:
There is a typo in the icon source path, should be: whats-new-in-aspnet-and-web-development-in-visual-studio-2012/_static/image1.png